scale-testing.mk

```diff
@@ -5,13 +5,19 @@
 AWS_REGION ?= us-east-2
 EKS_K8S_VERSION ?= 1.30
 
+VERSION ?= 0.8.1
+INTEGRATION_TESTS_PARALLEL ?= true
```

Comment:
Isn't this inherited from the main Makefile?

Reply:
It's not set in the Makefile. I thought I should set it explicitly here, so I don't have to set it each time. I can remove if needed.